examples/basic_gazelle/printlinks_test.go

```diff
@@ -12,16 +12,35 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package roll
+package main
 
 import (
+	"strings"
 	"testing"
-)
 
-func TestGenerateNumber(t *testing.T) {
-	result := generateNumber()
+	"golang.org/x/net/html"
+)
 
-	if result == "" {
-		t.Error("got an empty string")
+func TestParseAndGetLinks(t *testing.T) {
+	r := strings.NewReader(`<!DOCTYPE html>
+<html>
+  <head>
+	  <title>example</title>
+	</head>
+	<body>
+	  <a href="example.com/a">A</a><br>
+		<a href="example.com/b">B</a>
+	</body>
+</html>
+`)
+	node, err := html.Parse(r)
+	if err != nil {
+		t.Fatal(err)
+	}
+	links := getLinks(node)
+	got := strings.Join(links, ",")
+	want := "example.com/a,example.com/b"
+	if got != want {
+		t.Fatalf("got %q; want %q", got, want)
```

Comment:
We could add a second external dep on `assert`. That also demonstrates the distinction between prod and test deps in build files.

Reply:
Good idea, done.